infrastructure/gatekeeper/README.md

```diff
@@ -0,0 +1,103 @@
+
+
+GATEKEEPER
+=====================================
+## GOALS 
+Kubernetes allows you to write policies on your cluster objects by means of admission controller webhooks which are executed everytime a cluster component is created or modified.
+OPA (Open Policy Agent) is an open source, general-purpose policy engine to write unifed policies across different applications. It exploits REGO language to write policies.
+Gatekeeper allows you to integrate easily OPA on kubernetes by adding:
+ - native kubernetes CRDs for instantiating the policy library (Constraints)
+ - native kubernetes CRDs for extending the policy library (ConstraintTemplates)
+
+
+## HOW TO INSTALL
+### Prerequisites 
+- minimun kubernetes version: 1.14
+- make sure you have cluster admin permissions
+#### Deploying a Release using Prebuilt Image
+Run the following command:
+
+kubectl apply -f https://raw.githubusercontent.com/open-policy-agent/gatekeeper/release-3.1/deploy/gatekeeper.y
+
+## HOW TO USE GATEKEEPER
```

Comment:
I suppose this is copied from the official guide. In that case, you should add a reference to the original documentation.

Additionally, I would avoid having the full yaml file inside the documentation, as it is also added as an external file. IMO here you could describe what the different manifests are for and have a pointer to the full yaml file.